### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by yusrah247**
## Task: `[6] Product Page using HTML`
- `A web design for an online fashion store using HTML tags only`

### CODE REVIEW

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+	<div>
+		<section>
+			<nav>
+			<table width="100%">
+				<tr><th> <a href=""><h3>About us</h3></a></th>
+					<th><a href=""><h3>Men Wears</h3></a></th>
+					<th><a href=""><h3>Women Wears</h3></a></th>
+					<th> <a href=""><h3>Baby Wears</h3></a></th>
+				</tr>
```
Comment (adaugochi) [inline submissions/yusiratArogundade/introductionToHtml/index.html:28]: Remove new line spaces

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+
+			</table>
+
```
Comment (adaugochi) [inline submissions/yusiratArogundade/introductionToHtml/index.html:32]: same here

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+			<li><a href="">Toddler Girl Princess Bridesmaid Gown</a></li>
+		</ul>
+		
+	</div>
+	<hr>
+
+	<div>
+		<footer>
+			<table width="50%">
+				<tr><th><img src="email.png"><h2>ayusirat@gmail.com</h2></th>
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:118]: In the instructions for this task, you were asked to use only 2 product images but you have 5 images. Remove the extra images and delete the unused ones. Check [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7) for the instruction

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+			<li><a href="">Kids Girls Embroidery Wedding Costume Blue Tulle Party Dress</a></li>
+			<li><a href="">Toddler Girl Princess Bridesmaid Gown</a></li>
+		</ul>
+		
+	</div>
+	<hr>
+
+	<div>
+		<footer>
+			<table width="50%">
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:117]: Remove the table styling for the `width`

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+			<table>
+				<tr>
+					<th><img src="baby.png"></th>
+					<th><table>
+						<tr>
+							<th><h2>Baby Girl Ankara One Hand A-Cut Dress</h2></th>
+						</tr>
+						<tr>
+							<th><h3>Descriptiion</h3>
+								<ul align="left">
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:51]: Remove all the styling to `align` on this file

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+									<li>Beautiful African prints and soft cotton combo</li>
+									<li>Smart and simple</li>
+									<li>Classy hand crafted stylish designs for infants and  kids.</li>
+									<li>Very affordable, breathable and comfortable to wear in hot/dry seasons</li>
+									<li>It can be adjusted to fit as desired as it can suit a wide range of body length and sizes. </li>
+									<li>The materials are genuine DaViva and Woodin prints.</li>
+									<li>Durable, long lasting, and machine washable.</li>
+								</ul>
+							</th>
+							<tr><th><h4 align="left"><strong>Price &#8358; 4,500</strong></h4></th></tr>
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:61]: same as above

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+			<table>
+				<tr>
+					<th><img src="baby2.png"></th>
+					<th><table>
+						<tr>
+							<th><h2>Flower Girls Pleated Dress </h2></th>
+						</tr>
+						<tr>
+							<th><h3>Descriptiion</h3>
+								<ul align="left">
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:83]: same as above

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+									<li>Soft Material and first class workmanship.</li>
+									<li>Loose and comfortable</li>
+									<li>Daily Wear, School, Party Costume, Best gift for Girl</li>
+									<li>Material: Polyester + Cotton Blend</li>
+									<li>Sleeve: Sleeveless</li>
+								</ul>
+
+							</th>
+							<tr><th><h4 align="left"><strong>Price &#8358; 5,500</strong></h4></th></tr>
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:92]: same as above

File: submissions/yusiratArogundade/introductionToHtml/index.html [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+	<div>
+		<footer>
+			<table width="50%">
+				<tr><th><img src="email.png"><h2>ayusirat@gmail.com</h2></th>
+					<th><img src="call.png"><h3>07035703937</h3></th>
+				</tr>
+			</table>
+			<hr>
+			<p align="center">Copyright &copy; Indulgence Fashion Hub</p>
```
Comment (ToluAkin) [inline submissions/yusiratArogundade/introductionToHtml/index.html:123]: Remove align here

**Pull request closed without merging**